extensions/keycloak/deployment/src/main/java/io/quarkus/keycloak/KeycloakConfig.java

```diff
@@ -262,34 +262,34 @@
      * requests authenticated by the bearer token. This is set to false by default, however for improved security, it is
      * recommended to enable this. See Audience Support for more details about audience support
      */
-    @ConfigItem(name = "verify-token-audience", defaultValue = "false")
+    @ConfigItem
     boolean verifyTokenAudience;
 
     /**
      * If set to true will turn off processing of the access_token query parameter for bearer token processing.
      * Users will not be able to authenticate if they only pass in an access_token
      */
-    @ConfigItem(name = "ignore-oauth-query-parameter", defaultValue = "false")
+    @ConfigItem(name = "ignore-oauth-query-parameter")
     boolean ignoreOAuthQueryParameter;
```

Comment:
This one doesn't work and the property ends up being `o-auth`? I thought David had handled this case?

Reply:
Yes exactly. It ends up being `o-auth`. I think only `JBoss` is handled differently.